src/auth/authentication.ts

```diff
@@ -120,10 +121,13 @@ const authApiKey = async (request: FastifyRequest, apiKey: string) => {
       code: APIErrorCode.AUTH_ERROR
     });
   }
-  key.lastUsedAt = new Date();
   request.requestContext.set('apiKey', key);
```

Comment:
Note that the key in memory won't have the up-to-date value.

Reply:
Yes.